test/addurl_test.go

```diff
@@ -69,8 +69,10 @@ TAG ` + tag
 	})
 	defer os.RemoveAll(buildDir)
 
-	err := runRockerBuildWdWithOptions(buildDir, "-cache-dir", cacheDir)
-	assert.Nil(t, err, "no difference in downloaded and added files")
+	err := runRockerBuildWd(buildDir, "-cache-dir", cacheDir)
+	if err != nil {
```

Comment:
Because when it fails it continues function execution, while it's not a desired behavior. `assert.NoError` can be used if there is only one assertion in a function.